### PR TITLE
feat: support for .jsonc and .yml files

### LINKED ISSUE #1: Add support for deno.jsonc
The `deno.jsonc` is used by many projects in the Deno community, and is mentioned as a [supported extension in the Deno Manual](https://deno.land/manual@v1.26.2/getting_started/configuration_file). 

Unfortunately, this module's current API assumes all project will be using `deno.json`, rendering it effectively useless for anything else. Deno's Standard Library [has a nice JSONC module](https://deno.land/std/encoding/jsonc.ts?source), that could help make this a pretty simple fix.

If I can fit it in my schedule I'll put together a quick PR to fix this issue.

Thanks!

### PR DESCRIPTION
**Pull request by boywithkeyboard**
This pull request adds support for `deno.jsonc` or a separate `githooks.json`, `githooks.jsonc`, `githooks.yml`, or `githooks.yaml` file.

closes #1

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "recommendations": [
-    "denoland.vscode-deno"
+    "denoland.vscode-deno",
+    "stackbreak.comment-divider"
```
Comment (igorbrasileiro) [inline .vscode/extensions.json:4]: I don't know if it's valid to add this.

Comment (boywithkeyboard) [reply .vscode/extensions.json:4]: I can remove it, I used it only for the comment dividers.

Comment (boywithkeyboard) [reply .vscode/extensions.json:4]: What do you think apart from that? Is it okay to rename the `setupGithooks` function to `setup`? The module is used as CLI anyways, right?

**Comment by boywithkeyboard** [comment]
@igorbrasileiro what do you think?

I've renamed the `setupGithooks` function to `setup`, but this shouldn't affect anyone using this module since it is primarily used as a CLI anyway. So **for anyone using this module via TypeScript, this is a breaking change**.

**Pull request merged**